apps/demo/.env

```diff
@@ -3,27 +3,18 @@
 # Copy and configure the variables below in your own `.env.local` file
 #####
 
-# Whether to import @h5web/app's built files instead of its source files
-REACT_APP_DIST=
-
```

Comment:
Removed in favor of `import.meta.env.PROD` -- note that Vite is able to build the demo without using the distributed JS bundles of the packages. `PROD` is only used to import the distributed styles, as for some reason, global styles are not processed when building for production. I'm still investigating this.